### ISSUE TITLE
Port forwarding not working in Codespaces after updating to latest Evidence version (14.0)

### ISSUE THREAD

**Issue by aaronsteers** [evidence-dev/evidence#690]
## TL;DR

Issue can be resolved by setting `dev` script in `package.json` to `'evidence dev --host 0.0.0.0 --open /'`.

## Original issue description

I noticed this and thought it was just an issue on my side - then recently was confirmed by another developer. It looks like the bump to newer version of SvelteKit might have changed how ports are exposed.

Do we need to use the `--host` option or something similar to expose the port to the network in Evidence 14.0.0? Or has anyone else gotten port forwarding working again in Codespaces?

Sample project here:

- PR link: https://github.com/dbt-labs/jaffle-shop-template/pull/8/files
- Codespace for testing: https://github.com/codespaces/new?hide_repo_select=true&ref=bump-evidence-to-v14&repo=612676159

Launching a new Codespace with the link above should allow repro using steps:

```
cd reports
npm install
npm run dev
```

Redirect URL looks something like `https://auth.preview.app.github.dev/authenticate-codespace/aaronsteers-cuddly-meme-pj5ggpx559qc79jp?path=/true&visibility=private&port=3000&cid=100276c85f7a93f0454a737f93e9c6d3`

Which lands at `https://aaronsteers-cuddly-meme-pj5ggpx559qc79jp-3000.preview.app.github.dev/true`. (I tried removing "/true" from the URL with no change.)

Final result in codespaces is:

> ![image](https://user-images.githubusercontent.com/18150651/225046821-001e2431-77fe-471e-a0d6-04bc48d543a6.png)

**Comment by hughess** [comment]
@aaronsteers I was able to resolve the 502 errors on my end by changing the dev command in package.json to `evidence dev --host 0.0.0.0`

I've found that codespaces still doesn't recognize the port consistently though.

**Comment by aaronsteers** [comment]
Cool! That seems to have worked!

I updated the PR and the port is forwarded correctly now.

The only remaining issue I see is the '/true' suffix at the end of the path. Do you know anything about that? It's easy enough to remove manually for now during dev/test cycles, so not a major issue.

> I've found that codespaces still doesn't recognize the port consistently though.

I didn't have any issue - at least not when starting fresh in a new container.

**Comment by archiewood** [comment]
`evidence dev --open / --host 0.0.0.0`?

**Comment by hughess** [comment]
adding that slash to `--open` worked for me to fix the '/true' suffix!

**Comment by aaronsteers** [comment]
Confirmed for me too! 👍
